google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/MessageReceiver.java

```diff
@@ -36,8 +35,18 @@
   }
 
   /**
-   * Called when a message is received by the subscriber. The implementation must arrange for {@code
-   * reponse} to be set after processing the {@code message}.
+   * Accepts a reply, sending it to the service.
+   *
+   * <p>Both the interface and its method is named after the Java 8's {@code BiConsumer} interface
+   * to ease migration when we finally move.
```

Comment:
"when we finally move" -> "when we finally migrate to Java 8 and adopt its patterns"